comment/comment_test.go

```diff
@@ -124,6 +124,36 @@ func (test *TestCommentRepository) TestSaveCommentWithoutMarkup() {
 	assert.Equal(test.T(), rendering.SystemMarkupPlainText, comments[0].Markup)
 }
 
+func (test *TestCommentRepository) TestDeleteComment() {
+	defer cleaner.DeleteCreatedEntities(test.DB)()
+
+	t := test.T()
+	resource.Require(t, resource.Database)
```

Comment:
Place this as first call in the Test function. The function should do nothing if Database is not available.